modules/apps/portal-vulcan/portal-vulcan-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Portal Vulcan API
 Bundle-SymbolicName: com.liferay.portal.vulcan.api
-Bundle-Version: 14.0.1
+Bundle-Version: 15.0.0
```

Comment:
🤔 🤔 🤔 
Strange to me. I do not know why your changes are considered a breaking change for the baseline 🤷

Reply:
I can do again the baseline and see if it is a breaking or not :thinking: